data/rkhunter_checks.txt

```diff
@@ -0,0 +1,239 @@
+Checking for directory '/dev/ptyas'
```

Comment:
Would it perhaps be smarter to have a list of directories and a list of files? These could then also be consumed by other genact modules later on. It's also fine if you want to just play these files from the top for now, your choice.

Reply:
I'll have a look tomorrow.

Reply:
Thought about this and having files and directories in one file would be difficult to differentiate. If we decide to have normalized data for the Linux filesystem, I would create one file for directories and one for regular files. Maybe this could be done if another module needs this data?